src/NServiceBus.Core.Tests/OpenTelemetry/MeterTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace NServiceBus.Core.Tests.OpenTelemetry;
+
+using System.Diagnostics.Metrics;
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+using Particular.Approvals;
+
+[TestFixture]
+public class MeterTests
+{
+    [Test]
+    public void Verify_MeterAPI()
+    {
+        var meterTags = typeof(MeterTags)
+            .GetFields(BindingFlags.Public | BindingFlags.Static)
+            .Where(fi => fi.IsLiteral && !fi.IsInitOnly)
+            .Select(x => x.GetRawConstantValue())
+            .ToList();
```

Comment:
Guard against different platforms, .NET versions, bitness-es, or compilation randomness enumerating these in any other order.
```suggestion
            .OrderBy(value => value)
            .ToList();
```